### ISSUE TITLE
[Proposal] Смена парадигмы по поводу разгерметизаций

### ISSUE THREAD

**Issue by Deahaka** [TauCetiStation/TauCetiClassic#11670]
Обсуждение в теме https://forum.taucetistation.org/t/obsuzhdenie-nerfy-skafandrov/39765
#### Подробное описание проблемы
Текущая концепция заключается примерно в этих утверждениях:
1. Мы наказываем плохих игроков.
2. Если кто-то вызывает разгерметизацию, то он плохой игрок.
3. Если плохой игрок пользуется преимуществами, которые остальные не могут получить, эти преимущества нужно убрать.
4. Если преимущество нельзя убрать, его нужно нивелировать.

Успех политики заключается в НАБЛЮДАЕМОМ:
1. Снижении колличества разгерметизаций
2. Гарантия смерти тех, кто разгерметизирует.
#### Что должно было произойти
Скорость в ригах занерфлена, защита ригов занерфлена, магбутсы занерфлены, комбат бутсы занерфлены > снижение колличества разгерметизаций и невыгодность разгерм тем, кто имеет выбор делать перепады давления на станции или не делать.
### Нюка не умирает в разгермах, умирает только экипаж
![https://forum.taucetistation.org/t/obsuzhdenie-nerfy-skafandrov/39765/9](https://github.com/TauCetiStation/TauCetiClassic/assets/96499407/0da4f2ca-73a2-4eb6-9ea2-04cb45131c7e)
### Опрос игроков об эффективности данной стратегии
![0000000000000000000000000000000000](https://github.com/TauCetiStation/TauCetiClassic/assets/96499407/03335449-61f4-445e-a488-28ec7729b061)

30% считают что это успех, 10% считают что успех скоро прийдёт, 45% считают что это провал.
#### Что делать?
Перестать следовать текущей парадигме о разгерметизациях. Взять курс на упрощение вступления и ведения боя в условиях низкого давления для всего экипажа станции, а также увеличения возможностей защититься ригом от всего обычного оружия что работает в разгерме для всех. Это может очень революционная идея, но текущий путь не ведёт никуда кроме как к игре в одни ворота (разгермоёбы всегда победят любых неразгермоёбов)

**Comment by Tolerantboy** [comment]
Видел я билды, в которых разгерма максимально разнерфлена. Вся станция в дырах, экипажу похуй, ходят без скафандров мимо этих дыр в полу, и в ус не дуют, никаких ветров или давления. 
Выглядит максимально кринжово. 
Дырка в безопасной станции в супер-опасный космос мне кажется всегда должна восприниматься как должна — как угроза. 
Мне кажется, надо нерфить риги, но не всем, а конкретно нюкерам — у экипажа их мало и достать его тритору это отдельная палевная авантюра. 
Чтобы у нюкеров был выбор в экипировке: тонкие скафандры или толстая броня, не предназначенная для космоса. Скафандры положить в шаттл. 
Таким образом ещё должна, в теории, решиться проблема, что шаттл это самый оптимальный вариант прибытия, превалирующий над остальными и выкидывающий их на помойку. 
И появляется выбор: Вы либо можете двигаться по космосу, появляясь где угодно, заваливаясь в бриг из космоса по мете, и имея стабильное убежище в стане врага, НО больше рискуете в открытых столкновениях, если экипаж таки добрался до оружия и своих лакшери скафандров НТ с нормальной защитой. 
ИЛИ вы можете отсосаться наконец от шаттла, променяв однако доступ к космосу на более надёжную защиту в виде брони и запиздившись с экипажем в кислородных условиях не как ссыкло.

**Comment by Tolerantboy** [comment]
> Не идёт речи о нерфе эффектов разгерметизации. Речь лишь о том, чтобы у экипажа была возможность быстро найти скафандр, быстро его надеть и он защищал не как эрвос.

Но ведь ERVOC это, вроде, и есть те самые гражданские скафандры, прикол которых в том, что валяются на каждом углу. Может добавить больше ящиков с эрвоками? Сделать их настенными, чтобы их было больше где напихать? Сделать, чтобы они надевались быстрее ригов?

**Comment by Deahaka** [comment]
Да, именно в таком ключе и предлагаю думать. Замедление, которого меньше у обычных скафандров (вроде) нужно выравнять, а то и сделать меньше по сравнению с скафандрами-ригами. Надевать их нужно быстрее, но не только их, а и вообще все скафандры.

**Comment by RONSI7** [comment]
Лучше наоборот делать так, что при разгерметизации страдали вообще всех без исключений. Ускорить время умерщвления куклы в условиях отсутствия атмосферы в два раза от нынешней, сделать риги еще картоне, но в тоже время повысить их доступность и количество для всех/удешевить их покупку.

Все это приведет к тому что устраивать перестрелки в зоне отсутствия давления или открытом космосе станет нерациональным, ведь какая разница сколько у тебя оружия и химикатов в крови, если 2-3 шальные пули дырявят твой риг и тебе жопа.

Риги в моем понимании должны быть средством перемещения по космосу, а не какой либо броней.

**Comment by Tolerantboy** [comment]
> сделать риги ещё картонней

Чтобы шахтёры с вертухи получали за щёку от мобцов. Они и так 99% смертность и "Борьба с монстрами" это обычно убегание от них, роняя кал, а если уж выпал сингулярный невидимка, то вообще сворачивайте экспедицию, ресурсов сегодня больше не будет.

**Comment by KatoMarika** [comment]
С учетом нерфа космоса от Бетеля актуально еще? @Deahaka

**Comment by Deahaka** [comment]
Да тут совсем другая идея. Может когда-нибудь придёт избранный и сделает всё как надо
